src/store.ts

```diff
@@ -1,4 +1,4 @@
-import { atom, selector, useRecoilState, useRecoilValue } from 'recoil'
+import { atom, selector, SetterOrUpdater, useRecoilState, useRecoilValue, useSetRecoilState } from 'recoil'
```

Comment:
The [explain like I'm 5](https://www.youtube.com/watch?v=U9XStcquQyY) video on the recoil home page couldn't have been more obscure...

Reply:
Sorry to hear that. When I was picking a state manager I watched https://www.youtube.com/watch?v=P95DuIBwnqw&t=50s

Reply:
Luckily the [other video](https://www.youtube.com/watch?v=_ISAA_Jt9kI&feature=emb_logo) on their home page was much better: